src/App.scss

```diff
@@ -27,3 +27,9 @@ iframe {
 .card-image img {
   object-fit: contain;
 }
+
+.image {
+  width: 80%;
+  margin: 0 auto;
+  margin-top: 10%;
+}
```

Comment:
it's redundant

Reply:
can you be more specific? I received a comment to change the image a bit.